sdk/core/Test-Setup.ps1

```diff
@@ -57,7 +57,8 @@ if ($packageName -eq "azure_core_amqp") {
     # now that the Test broker has been built, launch the broker on a local address.
     $env:TEST_BROKER_ADDRESS = 'amqp://127.0.0.1:25672'
 
-    $remoteJob = Start-Process $workingDirectory"/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe" -ArgumentList { "$env:TEST_BROKER_ADDRESS, /headless" }
+    $brokerExecutable = "$workingDirectory/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe"
+    $remoteJob = Start-Process $workingDirectory/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe -ArgumentList { "$env:TEST_BROKER_ADDRESS, /headless" }
```

Comment:
This won't work either. Inside a double quoted string you need:

```suggestion
    $remoteJob = Start-Process $workingDirectory/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe -ArgumentList { "${env:TEST_BROKER_ADDRESS}, /headless" }
```

Or it won't evaluate properly. Basically, you're limited to `A-Za-z0-8_` or close to that.